packages/contract/package.json

```diff
@@ -11,7 +11,7 @@
   "version": "4.1.10",
   "main": "index.js",
   "scripts": {
-    "compile": "sh -c \"mkdir -p ./dist\" && browserify ./index.js -o ./dist/truffle-contract.js && uglifyjs ./dist/truffle-contract.js -o ./dist/truffle-contract.min.js",
+    "compile": "sh -c \"mkdir -p ./dist\" && browserify --debug ./index.js -o ./dist/truffle-contract.js && uglifyjs ./dist/truffle-contract.js -o ./dist/truffle-contract.min.js",
```

Comment:
but then it's not a production build? is that a problem? would like a second opinion... @CruzMolina? @eggplantzzz?

Reply:
@haltman-at want to split the contract stuff out into a separate PR so we can at least get the other packages updated while we hem and haw?

Reply:
OK, I've combined this with the use of `exorcist` per @CruzMolina's suggestion.